indico/modules/events/abstracts/controllers/abstract_list.py

```diff
@@ -95,6 +96,14 @@ class RHAbstractList(DisplayAbstractListMixin, RHAbstractListBase):
     template = 'management/abstract_list.html'
     view_class = WPManageAbstracts
 
+    def _render_template(self, **kwargs):
+        abstract_files_count = (AbstractFile.query
+                                .filter(AbstractFile.abstract.has(event=self.event, is_deleted=False))
+                                .count())
+        can_download_attachments = self.event.cfa.allow_attachments and abstract_files_count != 0
```

Comment:
that `!= 0` is kind of redundant... also, I think we want `or` here - if attachments are enabled or there are some abstracts with an attachment, the button should be visible